### PR TITLE
Fix broken URL for "Try TiKV and TiDB"

### PR DESCRIPTION
**Pull request by lsampras**
###  What have you changed?
The link for `Try TiKV and TiDB` under [Try TiKV](https://github.com/tikv/tikv#testing-deployment) in  README leads to `404`.

Updated the URL to point to [this](https://tikv.org/docs/3.0/tasks/try/) instead.

###  What is the type of the changes?

- Misc (other changes)

###  How is the PR tested?
- No code

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
No

###  Does this PR affect `tidb-ansible`?
No

###  Refer to a related PR or issue link (optional)
#6688

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ We provide multiple deployment methods, but it is recommended to use our Ansible
 
 ### Testing deployment
 
-- [Try TiKV and TiDB](https://github.com/pingcap/docs/blob/master/dev/how-to/get-started/deploy-tidb-from-docker-compose.md)
+- [Try TiKV and TiDB](./docs/how-to/get-started/deploy-using-docker-compose.md)
```
Comment (dcalvin) [inline README.md]:
```suggestion
- [Try TiKV and TiDB](https://tikv.org/docs/3.0/tasks/try/)
```

Comment (dcalvin) [reply README.md]: We are going to remove the docs folder soon because we don't maintain it.

Comment (lsampras) [reply README.md]: OK updating it..

**Comment by lsampras** [comment]
I did a force-push to replace a non-signed commit.
but looks like something else broke..

Can I re-run the tests?

**Comment by siddontang** [comment]
I merged the master once and ran `/test` explicitly twice, then passed the CI finally. (I worry it may fail soon :sob: ) 

I think our CI is very very unstable, please pay more attention to it. 

/cc @zhangjinpeng1987 @BusyJay @mahjonp @zhouqiang-cl

**Pull request merged**